### PR TITLE
Add input JSON

### LINKED ISSUE #496: Handle JSON input in content manager
**Node.js version**: 8.9.3
**npm version**: 5.6.0
**Strapi version**: 3.0.0-alpha.8.3
**Operating system**: Ubuntu 16.04 x64

Additional details -
**Database**: PostgreSQL
**Database version**: 9.6.6

Values in JSON fields appear as `Map {"<key>": <value> }`. For example 

Value saved through UI - `{"a": 5, "b": 10}`
Value persited in database - `{"a": 5, "b": 10}`
Value displayed in UI - `Map { "a": 5, "b": 10 }`

### PR DESCRIPTION
**Pull request by soupette**
My PR is a:
💅 Enhancement
🚀 New feature

Main update on the:
Plugin: `content-manager`

Created Input JSON
input json

Fixes #496

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
@lauriejim In case you guys weren't aware as of MySQL 5.7 and MariaDB 10.2 both now support the json data type (allows for better storage of JSON). I feel that requirements should be put into place that versions that match are newer than these two should be in place and that storage of json data should use this type.

*see the following docs*:
https://mariadb.com/kb/en/library/json-data-type/
https://dev.mysql.com/doc/refman/5.7/en/json.html

As for Postgres, further information is needed to determine the best use case here, I believe based on the following doc that the `json` data type is supported in postgres: https://www.postgresql.org/docs/9.4/static/datatype-json.html

As for mongo, you guys are the pros there *shrug*

**Comment by derrickmehaffy** [comment]
@Aurelsicoko @lauriejim There is an issue with json on mysql (possibly, with mongo as well) 

see: https://github.com/strapi/strapi/issues/1406

**Comment by derrickmehaffy** [comment]
related issue to PR: https://github.com/strapi/strapi/issues/496

**Comment by lauriejim** [comment]
@soupette Can you ping us when this PR is ready for review ?

File: packages/strapi-admin/admin/src/translations/en.json [hunk 1]
```diff
@@ -122,6 +122,7 @@
   "components.Input.error.attribute.sameKeyAndName": "Can't be equal",
   "components.Input.error.validation.minSupMax": "Can't be superior",
   "components.Input.error.custom-error": "{errorMessage} ",
+  "components.Input.error.validation.json": "This is not a JSON",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/en.json]: `This doesn't match with the JSON format`?

File: packages/strapi-plugin-content-manager/admin/src/utils/formValidations.js [hunk 2]
```diff
@@ -50,6 +50,8 @@ export function checkFormValidity(formData, formValidations) {
 }
 
 function validate(value, validations) {
+  // console.log('v', value);
+  // console.log('vza', validations);
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/utils/formValidations.js]: Remove the logs

File: packages/strapi-admin/admin/src/translations/en.json (hunk 1 above)
Comment (derrickmehaffy) [reply packages/strapi-admin/admin/src/translations/en.json]: +1 to that @Aurelsicoko though I would use `This doesn't match the JSON format`

**Comment by lauriejim** [comment]
@derrickmehaffy This PR is only UI stuff. Will fix in an other PR this issue https://github.com/strapi/strapi/issues/1425

**Pull request merged**